<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyMusicPlayer</class>
 <widget class="QMainWindow" name="MyMusicPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>937</width>
    <height>491</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MyMusicPlayer</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralWidget{
border-image: url(:/img/backimg.jpg);
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="psPuse">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>390</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/play.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="psAddfile">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>390</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/file.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPrev">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>390</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/prev.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="psNext">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>390</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>50</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">border-image: url(:/img/next.png);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionaddFile">
   <property name="text">
    <string>添加文件</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
